main.js

```diff
@@ -890,11 +890,11 @@ function _select_Gateway() { // Check for Gateway used to
             profileName: "default",
             isProfileActive: true,
             level: {
-                1: ["Blackice_Tier1_Process_Blackice"],
-                2: ["Blackice_Tier1_Process_Blackice"],
-                3: ["Blackice_Tier1_Process_Blackice"],
-                4: ["Blackice_Tier1_Process_Blackice"],
-                5: ["Blackice_Tier1_Process_Blackice"],
+                1: ["Blackice_Tier1_Process_Blackice","Blackice_Tier1_Recruit_Blackice_Smith"],
+                2: ["Blackice_Tier1_Process_Blackice","Blackice_Tier2_Recruit_Assistant_Cryomancer"],

```

Comment:
I don't see why'd you put `Blackice_Tier2_Recruit_Assistant_Cryomancer`. `Blackice_Tier1_Process_Blackice` only needs rank 1 assets, so destroying some assets to build a higher quality one isn't always desirable.